src/lib.rs

```diff
@@ -136,7 +136,74 @@ pub fn select_coin_lowestlarger(
     inputs: &[OutputGroup],
     options: CoinSelectionOpt,
 ) -> Result<SelectionOutput, SelectionError> {
-    unimplemented!()
+    let mut accumulated_value: u64 = 0;
+    let mut accumulated_weight: u32 = 0;
+    let mut selected_inputs: Vec<usize> = Vec::new();
+    let mut estimated_fees: u64 = 0;
+
+    let mut sorted_inputs: Vec<_> = inputs.iter().enumerate().collect();
+    sorted_inputs.sort_by_key(|(_, a)| std::cmp::Reverse(a.value));
+
+    let mut target = options.target_value + options.min_drain_value;
+    let mut min_idx = 0;
```

Comment:
Before the start of the loop using [binary search](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.binary_search_by) would be more readable and easy to verify

Reply:
Ok, will fix it.